content/development-guide/api/task-scheduler.md

```diff
@@ -24,9 +23,9 @@ namespace SamplePlugin
 	{
 		private ITaskScheduler taskScheduler;
 
-		public Main (IDependencyContainer container, ITaskScheduler taskScheduler) : base ("Sample Plugin", container)
+		public Main (IDependencyContainer container, ITaskScheduler _taskScheduler) : base ("Sample Plugin", container)
 		{
-			this.taskScheduler = taskScheduler;
+			taskScheduler = _taskScheduler;
```

Comment:
Whoops! Sorry, I didn't knew that it's against your code style. I just saw this style on almost every single newly ported plugin's source code.

Reply:
Have a look again, it's `_taskScheduler = taskScheduler` ;)

Reply:
Oh, yeah. It's actually like that xD But I prefer `taskScheduler = _taskScheduler;` because we are going to use `_scheduler`, right? So I just move that `_` from one to the other one. I shouldn't reflect my code style to there, sorry :D